theme-dev/README.md

```diff
@@ -1 +1,19 @@
-FOX HOW DO YOU DO THIS????
+# Themer Documentation for the Themer Plugin
+* [License](https://github.com/GangsterFox/AliuFox-themes/blob/main/LICENSE)
```

Comment:
remove, as a software license this doesn't apply to documentation, creative commons is more appropriate (already added it to the repo)

Reply:
done